### PR TITLE
feat(plugins.md): add Leaflet.TileCorrection

### PR DESCRIPTION
**Pull request by z632896862**
add Leaflet.TileCorrection

### CODE REVIEW

**Review by johnd0e — APPROVED** [review]

**Comment by Falke-Design** [comment]
@z632896862 Btw. If you want to reach more people I suggest you to write on your Github page in english.

**Comment by z632896862** [comment]
thank you, I will do it later




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Leaflet/Leaflet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年11月9日(星期二) 下午4:30
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Leaflet/Leaflet] feat(plugins.md): add Leaflet.TileCorrection (PR #7760)





 
@z632896862 Btw. If you want to reach more people I suggest you to write on your Github page in english.
 
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Review by Falke-Design — APPROVED** [review]
- [x] tabs checked
- [x] Readme of repo is in english
- [x] All links are working  

@jonkoops can be merged

File: docs/plugins.md [hunk 1]
```diff
@@ -774,6 +774,15 @@ The following plugins change the way that tile layers are loaded into the map.
 			<a href="https://github.com/allartk">Allart Kooiman</a>
 		</td>
 	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/z632896862/Leaflet.TileCorrection">Leaflet.TileCorrection</a>
+		</td><td>
+			Allow tiles to be loaded in an diffent crs from map's and start at a custom zoom.
```
Comment (johnd0e) [inline docs/plugins.md:781]:
```suggestion
			Allow tiles to be loaded in an different crs from map's and start at a custom zoom.
```

English text at homepage also has typo:
> Waring：the projection of map's CRS should be same as the custom CRS

^^^^ Warning

**Review by johnd0e — CHANGES_REQUESTED** [review]

File: docs/plugins.md (hunk 1 above)
Comment (Falke-Design) [reply docs/plugins.md:781]: @z632896862 can you please update the text

**Comment by Falke-Design** [comment]
We added the plugin in the refactoring where we move the plugins into seperate files #7805

**Pull request closed without merging**